### PR TITLE
add KubeSphere-terraform.md

### PR DESCRIPTION
**Pull request by Forest-L**
Signed-off-by: “Forest-L <lilin@yunify.com>

### CODE REVIEW

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
I'd suggest re-name the file to `KubeSphere-terraform-zh.md`

File: sig-installation/guideline/KubeSphere-terraform-zh.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+# 使用Terraform一键自动安装K8s和Ks
```
Comment (FeynmanZhou) [inline sig-installation/guideline/KubeSphere-terraform-zh.md:1]: 对外的文档里不要用 ks，都需要写成 KubeSphere。

File: sig-installation/guideline/KubeSphere-terraform-zh.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+* 使用Terraform作用是不需要用户在iaas平台上单独创建机器，配置好参数之后由Terraform自动创建，实现一键自动化部署。
```
Comment (FeynmanZhou) [inline sig-installation/guideline/KubeSphere-terraform-zh.md:3]: 中英文之间，还有数字和中英文之间左右两边都需要空一格。

File: sig-installation/guideline/KubeSphere-terraform-zh.md (hunk 1 above)
Comment (Forest-L) [reply sig-installation/guideline/KubeSphere-terraform-zh.md]: fixd

File: sig-installation/guideline/KubeSphere-terraform-zh.md (hunk 2 above)
Comment (Forest-L) [reply sig-installation/guideline/KubeSphere-terraform-zh.md]: fixd

File: sig-installation/guideline/KubeSphere-terraform-zh.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    }
+  }
+}
+```
+
+### 4、install.sh 文件说明
+* 具体执行命令
+
+```
+curl -O -k https://kubernetes.pek3b.qingstor.com/tools/kubekey/kk
```
Comment (pixiake) [inline sig-installation/guideline/KubeSphere-terraform-zh.md:149]: This should be specified as the link to the release.

Comment (Forest-L) [reply sig-installation/guideline/KubeSphere-terraform-zh.md]: Fixd

**Pull request merged**